examples/asr/conf/hat/conformer/conformer_hat_bpe.yaml

```diff
@@ -0,0 +1,267 @@
+# It contains the default values for training a Conformer-Hybrid_Autoregressive_Transducer ASR model, large size (~120M) with Transducer loss and sub-word encoding.
+
+# Architecture and training config:
```

Comment:
Would you please add the link to HAT paper here and also to the description of the PR?

Reply:
done